web/libs/editor/src/components/AnnotationsCarousel/AnnotationsCarousel.tsx

```diff
@@ -86,26 +83,24 @@ export const AnnotationsCarousel = observer(({ store, annotationStore }: Annotat
       </Elem>
       {(!isLeftDisabled || !isRightDisabled) && (
         <Elem name="carousel-controls">
-          <Elem
-            tag={Button}
-            name="nav"
+          <Button
             disabled={isLeftDisabled}
-            mod={{ left: true, disabled: isLeftDisabled }}
             aria-label="Carousel left"
-            onClick={(e: MouseEvent) => !isLeftDisabled && updatePosition(e, true)}
+            size="small"
+            variant="neutral"
+            onClick={(e) => !isLeftDisabled && updatePosition(e, true)}
           >
-            <Elem name="arrow" mod={{ left: true }} tag={IconChevron} />
-          </Elem>
-          <Elem
-            tag={Button}
-            name="nav"
+            <IconChevron style={{ rotate: "-90deg" }} />
+          </Button>
+          <Button
             disabled={isRightDisabled}
-            mod={{ right: true, disabled: isRightDisabled }}
             aria-label="Carousel right"
-            onClick={(e: MouseEvent) => !isRightDisabled && updatePosition(e, false)}
+            size="small"
+            variant="neutral"
+            onClick={(e) => !isRightDisabled && updatePosition(e, false)}
           >
-            <Elem name="arrow" mod={{ right: true }} tag={IconChevron} />
-          </Elem>
+            <IconChevron style={{ rotate: "90deg" }} />
```

Comment:
now that the icon work is done we should be able to use `IconChevronLeft` and IconChevronRight` instead of rotating `IconChevron`